src/page-analysis/index.js

```diff
@@ -4,6 +4,10 @@
 export const searchableTextFields = [
     'title',
     'extractedMetadata.title',
+    'extractedMetadata.Title',
+    'extractedMetadata.Author',
+    'extractedMetadata.Subject',
+    'extractedMetadata.Keywords',
```

Comment:
Good idea to make these searchable. We may want to harmonise the metadata of html pages and pdfs; having `title` for one and `Title` for the other looks awful. The whole choice of fields and names is up for revision though, but we could do all this after this PR.